internal/oem/oem.go

```diff
@@ -138,6 +135,10 @@ func init() {
 			},
 		},
 	})
+	configs.Register(Config{name: "cloudstack",
+		fetch:             cloudstack.FetchConfig,
+		defaultUserConfig: types.Config{Systemd: types.Systemd{Units: []types.Unit{userCloudInit("CloudStack", "cloudstack")}}},
```

Comment:
coreos-cloudinit does not support CloudStack. This default user config should be removed.